moderator/settings/base.py

```diff
@@ -1,81 +1,92 @@
+STATIC_ROOT = path('static')
+STATIC_URL = '/static/'
 
-DATABASES = {
-    'default': {
-        # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'ENGINE': 'django.db.backends.sqlite3',
-        # Or path to database file if using sqlite3.
-        'NAME': 'moderator-db',
-        # The following settings are not used with sqlite3:
-        'USER': '',
-        'PASSWORD': '',
-        # Empty for localhost through domain sockets or
-        # '127.0.0.1' for localhost through TCP.
-        'HOST': '',
-        'PORT': '',  # Set to empty string for default.
-    }
-}
+ADMINS = (
+    ('Mozilla Reps Devs', 'reps-dev@mozilla.com'),
+)
+MANAGERS = ADMINS
 
 # Hosts/domain names that are valid for this site; required if DEBUG is False
 # See https://docs.djangoproject.com/en/1.5/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['https://moderator.mozilla.org']
 
 # Local time zone for this installation. Choices can be found here:
 # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
 # although not all choices may be available on all operating systems.
-# In a Windows environment this must be set to your system time zone.
-TIME_ZONE = 'America/Chicago'
-
-# Language code for this installation. All choices can be found here:
-# http://www.i18nguy.com/unicode/language-identifiers.html
-LANGUAGE_CODE = 'en-us'
-
-SITE_ID = 1
-
+TIME_ZONE = 'UTC'
+# If you set this to False, Django will not use timezone-aware datetimes.
+USE_TZ = True
 # If you set this to False, Django will make some optimizations so as not
 # to load the internationalization machinery.
-USE_I18N = True
-
+USE_I18N = False
 # If you set this to False, Django will not format dates, numbers and
 # calendars according to the current locale.
-USE_L10N = True
-
-# If you set this to False, Django will not use timezone-aware datetimes.
-USE_TZ = True
-
-PROJECT_DIR = os.path.dirname(__file__)
-
-# Absolute filesystem path to the directory that will hold user-uploaded files.
-# Example: "/var/www/example.com/media/"
-MEDIA_ROOT = ''
-
-# URL that handles the media served from MEDIA_ROOT. Make sure to use a
-# trailing slash.
-# Examples: "http://example.com/media/", "http://media.example.com/"
-MEDIA_URL = '/media/'
+USE_L10N = False
 
-# Absolute path to the directory static files should be collected to.
-# Don't put anything in this directory yourself; store your static files
-# in apps' "static/" subdirectories and in STATICFILES_DIRS.
-# Example: "/var/www/example.com/static/"
-STATIC_ROOT = os.path.join(PROJECT_DIR, 'static')
+SITE_ID = 1
 
-# URL prefix for static files.
-# Example: "http://example.com/static/", "http://static.example.com/"
-STATIC_URL = '/static/'
+# Instruct session-csrf to always produce tokens for anonymous users
+ANON_ALWAYS = True
 
 # Additional locations of static files
-STATICFILES_DIRS = (os.path.join(PROJECT_DIR, 'moderate/static'),)
+# STATICFILES_DIRS = (os.path.join(PROJECT_DIR, 'moderate/static'),)
```

Comment:
Let's remove this commented line if we are not using extra ``STATICFILES_DIRS``.